plottable.css

```diff
@@ -164,6 +164,14 @@ svg.plottable {
   stroke-width: 1px;
 }
 
+.plottable .drag-line-layer.enabled.vertical line.drag-edge {
+  cursor: ew-resize;
+}
+
+.plottable .drag-line-layer.enabled.horizontal line.drag-edge {
+  cursor: ns-resize;
+}
```

Comment:
Using a hand cursor would also be general for non-90 degree lines, but I can see the argument for being more specific on 90-degree lines.  Can we get designer input on this?